picard/cluster.py

```diff
@@ -94,6 +100,10 @@ def remove_file(self, file):
         self.item.remove_file(file)
         if not self.special and self.get_num_files() == 0:
             self.tagger.remove_cluster(self)
+        self.update_metadata_images()
+        if self.related_album:
```

Comment:
These same 3 lines have been repeated thrice, we can make a private function instead?